modules/op-release-notes-1-8.adoc

```diff
@@ -50,8 +238,141 @@ Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image s
+// link:https://github.com/tektoncd/pipeline/issues/5295(#5295)
+//Andrew Bayer @abayer
+
+
+* Before this update, if the `<NAME>` in `$params["<NAME>"]` or `$params['<NAME>']` contained a dot character (`.`),  any part of the name to the right of the dot was not extracted. For example, from `$params["org.ipsum.lorem"]`, only `org` was extracted.
++
+This update fixes the issue so that `$params` fetches the complete value. It also throws an error if `<NAME>` is not enclosed in single or double quotes. For example:
++
+* `$params.org.ipsum.lorem` is not valid and generates a validation error.
+* `$params["org.ipsum.lorem"]` and `$params['org.ipsum.lorem']` are valid and the entire value of `<NAME>`, `org.ipsum.lorem`, is extracted.
+// link:https://github.com/tektoncd/pipeline/pull/4880
+
+// .Triggers
+
+With this update, `Trigger` resources support custom interceptors and ensure that the port of the custom interceptor service is the same as the port in the `ClusterInterceptor` definition file.
+// link:https://github.com/tektoncd/triggers/pull/1383[(#1383)]
+// @savitaashture
+
+
+// .CLI
+
+* Before this update, the `tkn version` command for {tekton-chains} and Operator components didn't work correctly. This update fixes the issue so that the command works correctly and returns version information for those components.
+// https://github.com/tektoncd/cli/pull/1542
+// Piyush Garg @piyush-garg
+
+* Before this update, if you ran a `tkn pr delete --ignore-running` command and a pipeline run did not have a `status.condition` value, the `tkn` CLI tool produced a null-pointer error (NPE). This update fixes the issue so that the CLI tool does generate an error and correctly ignores pipeline runs that are still running.
+// https://github.com/tektoncd/cli/pull/1532
+// PRADEEP KUMAR @pradeepitm12
+
+* Before this update, if you used the `tkn pr delete --keep <value>` or `tkn tr delete  --keep <value>` commands, and the number of pipeline runs or task runs was less than the value, the command did not return an error as it should. This update fixes the issue so that the command correctly returns an error under those conditions.
+// https://github.com/tektoncd/cli/pull/1541
+// Shiv Pratap Verma @pratap0007
+
+* Before this update, if you used the `tkn pr delete` or `tkn tr delete` commands with the `-p` or `-t` flags together with the `--ignore-running` flag, the commands incorrectly deleted running or pending resources. This update fixes the issue so that these commands correctly ignore running or pending resources.
+// https://github.com/tektoncd/cli/pull/1578
+// Piyush Garg @piyush-garg
+
+// .Operator
+
+* With this update, you can configure {tekton-chains} by using the `TektonChain` custom resource. This feature enables your configuration to persist across upgrades, unlike the `chains-config` config map, which gets overwritten during upgrades.
+// no link provided
```

Comment:
Would someone provide a link to the issue? @sm43?

Reply:
https://issues.redhat.com/browse/SRVKP-2349

Reply:
https://github.com/tektoncd/operator/pull/955
https://issues.redhat.com/browse/SRVKP-2349

Reply:
this feature was implemented for a bug, so it's both, a bugfix and an enhancement 🙃